TMessagesProj/build.gradle

```
@@ -5,6 +5,7 @@ apply plugin: 'com.android.library'
 repositories {
     mavenCentral()
     google()
+    maven { url 'https://www.jitpack.io' }
```

```suggestion
    maven {
            url "https://www.jitpack.io"
            content {
                includeModule 'com.github.UnifiedPush', 'android-connector'
            }
        }
```

To avoid other library to be fetched from jitpack